packages/gallery/tsconfig-esm-dev.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "./tsconfig-esm.json",
+  "compilerOptions": {
```

Comment:
Maybe we should copy the existing compilerOptions and adding the es2017 on top.
These are settings I worked on to have an easier TS start. (noImplicitAny etc).
Otherwise the tsc will break the build with different "reasons" than the rest, WDYT?

Reply:
Thanks to Rita, I see the extend :)

Reply:
I added a seperate file and configuration beecause I didn't want to touch the existing ones. 
I was not sure why the `target` was configured to `es5`. 
If we can change it then this configuration file can be totally removed

Reply:
Maybe we can change it actually, the target-esm.json. to use ES2017 and not es5.
I'll test it locally and update :)